fizzbuzz/README.md

```diff
@@ -0,0 +1,12 @@
+# FIZZBUZZ SOLUTION MANUAL
+
+This is a simple instructional guide how to run my python code with a shell script. 
+
+## Directions
+
+1. Make sure you're in fizzbuzz/python/loganDolechek
+2. In your command line type ./sb.sh # 
+3. # being the number you want to input to the python code. 
+4. Press enter. 
+5. You should be prompted to input a number of your choosing. Do so & press enter. 
```

Comment:
Could you adjust your code to only ask for the number to be used in your fizzbuzz once?